### DISCUSSION TITLE
mutual badge gone after the last update

### DISCUSSION THREAD

**Discussion by alexevah** [AprilSylph/XKit-Rewritten#1142] [Help Desk] [answered]
The little handy mutual badge is not showing up anymore next to the blog's url on dashboard
https://imgbox.com/EtBE6pCa

**Comment by AprilSylph** [comment, marked as answer]
This is already fixed in v0.21.13. Google has approved this update for Chrome, but we're still waiting for Mozilla to approve it for Firefox.

Original issue: #1138
